File: Python/UberVan.py
from Car import Car

class UberVan(Car):
    TypeCarAccepted =[]
    SeatMaterial = []
    
    
    def __init__(self,license,driver,TypeCarAccepted,SeatMaterial):
        super().__init__(license,driver)
        self.TypeCarAccepted=TypeCarAccepted
        self.SeatMaterial=SeatMaterial

File: Java/UberVan.java
import java.util.Map;

class UberVan extends Car{
    Map<String, Map<String,Integer>> typeCarAccepted;
    Map<String> seatMaterial;



    public UberVan(String license, Account driver, Map<String, Map<String,Integer>> typeCarAccepted){
        super(licence, driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatMaterial = seatMaterial;
    
    }
}